src/libstd/sync/once.rs

```diff
@@ -103,8 +103,8 @@ unsafe impl Sync for Once {}
 #[stable(feature = "rust1", since = "1.0.0")]
 unsafe impl Send for Once {}
 
-/// State yielded to the [`call_once_force`] method which can be used to query
-/// whether the [`Once`] was previously poisoned or not.
+/// State yielded to [`call_once_force`]’s closure parameter. The state can be
+/// used to query the poisoned status of the [`Once`].
```

Comment:
Might prefer "poison status" here, since that feels like it would better imply "the status being poisoned" rather than "the status which has been poisoned".  Tiniest nit though.

Reply:
sounds reasonable! addressed in https://github.com/rust-lang/rust/pull/45429/commits/8aad14cbc350cabe5a4e84a4038f175093748eb4